### PR TITLE
Dishwasher

### PR DESCRIPTION
**Pull request by Fris0uman**
#### Summary
```SUMMARY: Content "Add dishwasher vehicle part"```

#### Purpose of change
Add a vehicle part that can batch clean non soft items

#### Describe the solution
Add a new vehicle part that can wash item without any soft material
Make it craftable
Add it to Luxury RV

#### Additional context
Can wash anything that has no soft material, so anything made only from steel, plastic, ceramic or any other non soft thing (or a combinaison of those) can go in and get washed.

### CODE REVIEW

File: data/json/vehicleparts/vehicle_parts.json [hunk 1]
```diff
@@ -813,6 +813,30 @@
...
+    "type": "vehicle_part",
+    "id": "dishwasher",
+    "name": "dishwasher",
+    "symbol": "d",
+    "color": "light_blue",
+    "broken_symbol": "#",
+    "broken_color": "light_blue",
+    "damage_modifier": 80,
+    "durability": 100,
+    "description": "A small dishwasher.  With detergent or soap, water, and some electrical power, you could clean your dishes...  Or something more usefull like a CBM dripping with zombie gore.  'e'xamine the tile with the dishwasher to use it.",
```
Comment (Night-Pryanik) [inline data/json/vehicleparts/vehicle_parts.json]:
```suggestion
    "description": "A small dishwasher.  With detergent or soap, water, and some electrical power, you could clean your dishes...  Or something more useful like a CBM dripping with zombie gore.  'e'xamine the tile with the dishwasher to use it.",
```

File: data/json/vehicleparts/vehicle_parts.json [hunk 2]
```diff
@@ -813,6 +813,30 @@
...
+    "epower": -400,
+    "size": 160,
+    "item": "dishwasher",
+    "difficulty": 3,
+    "location": "center",
+    "flags": [ "CARGO", "OBSTACLE", "DISHWASHER" , "COVERED", "TOOL_WRENCH", "ENABLED_DRAINS_EPOWER" ],
```
Comment (anothersimulacrum) [inline data/json/vehicleparts/vehicle_parts.json]:
```suggestion
    "flags": [ "CARGO", "OBSTACLE", "DISHWASHER", "COVERED", "TOOL_WRENCH", "ENABLED_DRAINS_EPOWER" ],
```

**Comment by Night-Pryanik** [comment]
Uhmm, sorry for pointing on errors on draft PR.

**Comment by Fris0uman** [comment]
No problem, I'm glad for the help. Draft or PR I would have missed the misspel of useful

**Comment by pjf** [comment]
Perfect timing for all those survivors [trying to wash CBMs in a clothes-washing machine](https://github.com/CleverRaven/Cataclysm-DDA/pull/32686)!

**Comment by chronicpayne** [comment]
Might this be more appropriate for gold, silver, platinum, and diamond as well?

I don't think I would use either a washing machine _or_ a dishwasher for jewelry, unless I had a lot to clean at once and it wasn't sentimental, but since both of these things could apply in CDDA it would make more sense to use a dishwasher - jets of hot water and steam would be much more effective than being agitated in a washer cycle.

Basic example https://youtu.be/GCaVpFS_-7g?t=29
More or less all you need is something to secure the items with so they dont drain with the waste water.  

I'd argue for no extra requirement at all though, since most jewelry is large enough that it would get caught on the filter anyway, and if that fails - this is in a vehicle - the water is designed to drain somewhere not terribly difficult to access.

**Comment by anothersimulacrum** [comment]
It's not checking for specific materials, just if it's soft or not. So I think Jewelry can be washed in it.

**Pull request merged**